exchange/validation.go

```diff
@@ -0,0 +1,50 @@
+package exchange
+
+import (
+	"fmt"
+	"github.com/prebid/prebid-server/openrtb_ext"
+)
+
+type bidResponseWrapper struct {
```

Comment:
It is a hard call. But as per below, not making the validate functions methods, then moving this back to `exchange.go` is the way to go.

Reply:
ohh now I see why you had to move it! Go's limitations on using a receiver outside of the file it is defined in. Ahh! 

I personally like `validation.go` to be a separate file. Been on my mind to suggest that for `endpoints/openrtb2/auction.go` as well. But if it's just two methods, I'd suggest for now, it would be better to include it in `exchange.go` itself (owing to consistency..?)